posts/2014-02-10-io-js-and-a-nodejs-foundation.md

```diff
@@ -0,0 +1,45 @@
+#io.js en een Node.js stichting
+#### De huidige staat van verzoening
+
+Vorige week nodigde Scott Hammond, CEO van Joyent, het [io.js TC][iojstc]
+(Technisch Committee) uit voor een gesloten vergadering waarin hij zijn
+intentie uitte om een node.js stichting te beginnen en zijn verlangen om io.js
+terug te krijgen in het node.js project.
+
+In de afgelopen maanden is io.js gegroeid to 23 actieve kernleden,
+verschillende _working groups_, 27 lokalisatie teams en is in staat geweest om
+met regelmaat kwaliteitssoftware te releasen dankzij de ondersteuning van een
+exceptionele community. We hebben dit weten te bereiken door een open
+management structuur die de community heeft doen herleven en meer
+_contributors_ heeft aangetrokken dan we ooit hebben gehad in de geschiedenis
+van node.js.
+
+Het enige wat io.js beter kan maken is om de vragen over de toekomonst van de 
+splitsing met node.js te kunnen laten rusten. We willen dit graag achter ons
+laten maar kunnen de vooruitgang die we hebben gmaakt niet opgeven, net zo min
+als onze principes en de open management die ons hier heeft gebracht.
+
+Gesprekken met Joyent blijven in gang. Zodra de stichting een technisch
+management model heeft zal er een _issue_ verschijnen op de io.js GitHub die
+zal worden beproken tijdens een publieke TC meeting. In diezelfde meeting die
+volgens de reeds opgebouwde open management regels verloopt zal er vervolgens
+ook over worden gestemd.
+
+Voor de community veranderd er niks. Blijf doorgaan met het inzenden van _pull
+requests_ naar [io.js][iojs], word lid van deen van de 27 [lokalisatie
+teams][iojsi18n], draag bij aan een van de io.js _working groups_
+([streams][streamswg], [website][websitewg], [evangelism][evangelismwg],
+[tracing][tracingwg], [build][buildwg], [roadmap][roadmapwg]), en blijf met het
+doorgaan van het adopteren van io.js in je applicaties. We zijn van plan om
+door te gaan met releasen en verbeteren van io.js zelfs al kan de naam op een
+dag verandere ☺
+
+[iojs]: https://github.com/iojs/io.js
+[iojstc]: https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#technical-committee
```

Comment:
This is the same link that was used in the original article, think it's best to keep it similar for consistency.